wremnants/datasets/datagroups.py

```diff
@@ -197,13 +169,6 @@ def excludeGroups(self, excludes):
         
         if len(self.groups) == 0:
             logger.warning(f"Excluded all groups using '{excludes}'. Continue without any group.")
-
```

Comment:
Same, I assume the purpose of this function that is being removed is now taken over by the other new one as I see in datagroups2016.py
It might be useful (unless the new function is fully equivalent) to run some local stress tests to make sure the correct behaviour happens in unusual situations where some datasets were excluded (as dealing with these cases was one of the purposes of this function)